### PR TITLE
UX-1523: Updated Radios with the new implementation + Added subtitle t…

### PR DESCRIPTION
**Pull request by patil-pratik-87**
Screenshot 2022-02-03 at 4 33 38 PM
Screenshot 2022-02-03 at 4 56 44 PM
Screenshot 2022-02-03 at 5 14 11 PM
Screenshot 2022-02-02 at 9 13 52 PM
Screenshot 2022-02-02 at 9 14 03 PM
Screenshot 2022-02-03 at 11 18 05 PM

### CODE REVIEW

File: src/app/core/components/BlockChooser.tsx [hunk 1]
```diff
@@ -69,11 +73,16 @@ const BlockChooser: FunctionComponent<Props> = ({ options = [], onChange }) => {
             key={option.id}
             onClick={handleOnChange(option.id)}
           >
-            <Radio data-testid={generateTestId(option.id)} checked={active} color="primary" />
+            <Radio
+              className={classes.radio}
+              data-testid={generateTestId(option.id)}
+              checked={active}
+            />
+            {/* <Radio data-testid={generateTestId(option.id)} checked={active} color="primary" /> */}
```
Comment (jruddell) [inline src/app/core/components/BlockChooser.tsx:81]: remove comment

File: src/app/core/components/SingleSelect.tsx [hunk 2]
```diff
@@ -133,6 +133,7 @@ const SingleSelect: ComponentType<Props> = React.forwardRef<HTMLElement, Props>(
             style={{ height: maxOptions ? getOptionsHeight(maxOptions) : 'initial' }}
           >
             <RadioFields
+              row
```
Comment (jruddell) [inline src/app/core/components/SingleSelect.tsx:136]: lets call this something more like `orientation` and accept a `row` or `column` value

File: src/app/plugins/kubernetes/components/infrastructure/clusters/ClusterUpgradeDialog.tsx [hunk 3]
```diff
@@ -69,20 +82,13 @@ const ClusterUpgradeDialog = ({ rows: [cluster], onClose }) => {
...
-        />
-        <FormControlLabel
-          value={UpgradeTypes.Patch}
-          control={<Radio color="primary" />}
-          label={`Patch - ${cluster.patchUpgradeRoleVersion}`}
-        />
-      </RadioGroup>
-    </>
+    <RadioFields
+      id="updgradeOptions"
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/infrastructure/clusters/ClusterUpgradeDialog.tsx:86]: call this something more specific as id needs to be unique. `clusterUpgrade_VersionOptions` or something like that would be better

File: src/app/plugins/kubernetes/components/infrastructure/clusters/ScaleWorkersPage.tsx [hunk 4]
```diff
@@ -143,7 +140,7 @@ const ScaleWorkers: FunctionComponent<ScaleWorkersProps> = ({
   const chooseScaleNum = (
     <ValidatedForm initialValues={cluster} onSubmit={onSubmit} elevated={false}>
       {!autoscalingEnabled && (
-        <div className={classes.formFields}>
+        <div>
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/infrastructure/clusters/ScaleWorkersPage.tsx:143]: is the div still needed?

File: src/app/plugins/kubernetes/components/infrastructure/clusters/ScaleWorkersPage.tsx [hunk 5]
```diff
@@ -229,31 +225,27 @@ const ScaleWorkers: FunctionComponent<ScaleWorkersProps> = ({
...
               icon: <FontAwesomeIcon size="2x" name="layer-plus" />,
-              description: 'Add worker nodes to the cluster',
+              description: 'Add worker node(s) to the cluster',
             },
             {
               id: 'remove',
               icon: <FontAwesomeIcon size="2x" name="layer-minus" />,
               title: 'Remove',
-              description: 'Remove worker nodes from the cluster',
+              description: 'Remove worker node(s) from the cluster',
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/infrastructure/clusters/ScaleWorkersPage.tsx:234]: 👍

File: src/app/plugins/kubernetes/components/infrastructure/clusters/ScaleWorkersPage.tsx [hunk 6]
```diff
@@ -53,17 +53,14 @@ const useStyles = makeStyles((theme: Theme) => ({
...
-  workerCount: {
-    margin: theme.spacing(4, 0),
-  },
   errorMessage: {
     padding: theme.spacing(1),
   },
-  fullWidth: {
-    width: '70% !important',
+  subtitles: {
+    margin: theme.spacing(2, 0),
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/infrastructure/clusters/ScaleWorkersPage.tsx:60]: Generally I dislike using margins for spacing things around, is the parent here a grid? if it is you can probably position things how you'd like with a gap vs using margins

File: src/app/plugins/kubernetes/components/virtual-machines/add/presets-step.tsx [hunk 7]
```diff
@@ -131,6 +131,7 @@ export const PresetsStep = ({ wizardContext, setWizardContext, onNext }) => {
           )}
         </div>
         <RadioFields
+          row
```
Comment (jruddell) [inline src/app/plugins/kubernetes/components/virtual-machines/add/presets-step.tsx:134]: Hmm I think a row is probably more common and would probably be considered a default layout.. should we switch to `orientation: 'row'` as the default value?

File: src/app/plugins/kubernetes/components/infrastructure/clusters/ClusterUpgradeDialog.tsx (hunk 3 above)
Comment (patil-pratik-87) [reply src/app/plugins/kubernetes/components/infrastructure/clusters/ClusterUpgradeDialog.tsx:86]: What case are we following for the ids..shouldn't it be camelCase? I will use a more relevant name though

File: src/app/plugins/kubernetes/components/infrastructure/clusters/ScaleWorkersPage.tsx (hunk 4 above)
Comment (patil-pratik-87) [reply src/app/plugins/kubernetes/components/infrastructure/clusters/ScaleWorkersPage.tsx]: nope.
removed

File: src/app/core/components/validatedForm/radio-fields.tsx [hunk 8]
```diff
@@ -9,13 +9,18 @@ import { ValidatedFormProps } from './model'
...
 interface StyleProps {
-  row: boolean
+  orientation: Orientation
 }
 const styles = (theme: Theme) => ({
   root: {
     display: 'grid',
-    gridAutoFlow: ({ row }: StyleProps) => (row ? 'column' : 'row'),
+    gridAutoFlow: ({ orientation }: StyleProps) =>
+      orientation === Orientation.Column ? 'row' : 'column',
```
Comment (jruddell) [inline src/app/core/components/validatedForm/radio-fields.tsx:23]: you're writing this backwards I think. Shouldn't a column be a column. if I say orientation is column I expect columns not rows

**Review by jruddell — APPROVED** [review]

File: src/app/core/components/validatedForm/radio-fields.tsx (hunk 8 above)
Comment (patil-pratik-87) [reply src/app/core/components/validatedForm/radio-fields.tsx]: I took the convention from the material-ui semantics. orientation = 'row' meaning horizontal radios. We can use the inverse as well similar to the grid semantics. Will make the change.

**Pull request merged**